src/core/workspace/folder-to-project.ts

```diff
@@ -8,6 +8,9 @@ import { formatResourceType, formatResourceKey, formatAsHexString } from "@s4tk/
 import { findGlobMatches, parseKeyFromTgi } from "#building/resources";
 import StringTableJson from "#stbls/stbl-json";
 import * as inference from "#indexing/inference";
+var sanitize = require("sanitize-filename");
```

Comment:
Use import syntax.

Reply:
If you're wondering about the few other edits that came with this fix, "This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export. ts(2497)"